imsim/stamp.py

```diff
@@ -203,10 +205,17 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
 
         logger.info('Object %d will use stamp size = %s',base.get('obj_num',0),image_size)
 
-        # Also the position
-        world_pos = galsim.config.ParseWorldPos(config, 'world_pos', base, logger)
-        self.sky_coord = world_pos
-        image_pos = None  # GalSim will figure this out from the wcs.
+        # Determine where this object is going to go:
+        # This is the same as what the base StampBuilder does:
```

Comment:
All looks good again to me.

Just curious, what happens if both image_pos and world_pos are present in the config, but are mutually inconsistent with the WCS?

Reply:
IIRC, you can do this, and it will respect your values of both, ignoring the absolute calibration of the wcs.  The Jacobian of the wcs would still be used where appropriate, but not used to place the objects on the stamp.